rules/optimize-android-builds-and-start-up-times/rule.md

```diff
@@ -10,18 +10,114 @@ authors:
   url: https://ssw.com.au/people/adam-cogan
 - title: Matt Goldman
   url: https://ssw.com.au/people/matt-goldman
+- title: Anton Polkanov
+  url: https://ssw.com.au/people/anton-polkanov
 related: []
 redirects:
 - do-you-optimise-your-android-builds-and-start-up-times
-
 ---
 
-Whereas iOS enforces AOT (ahead of time) compilation, Android supports (and uses by default) JIT (just in time) compilation, but AOT can be enabled on Android to improve performance.
+.NET MAUI provides several ways to optimize an Android application. Some of them complement each other while others can be mutually exclusive.
+It's important to understand what options you have at hands and how they affect your application.
 
 <!--endintro-->
 
-Xamarin also provides several other options for optimizing Android builds and improving performance. Additionally, you can use Android's new bundle format (rather than apk) so that smaller builds can be targeted to individual hardware specifications.
+## Intro
+
+When it comes to app optimizations, developers usually try to strike the right balance between:
+* app build time
+* app size
+* app performance (execution speed and memory)
+
+Quite often improvements in one area lead to degradation in another. In most cases it's a tradeoff which developers need to take depending on their circumstances. Different Debug and Release configurations partially address this problem, but may lead to configuration-specific bugs.
+
+.NET MAUI Android combines two worlds - native Android and .NET, which means that optimization can be performed in both worlds.
+
+## Native Layer optimizations
+
+These optimizations are not specific to .NET MAUI and are used in other cross-platform frameworks or native Android development.
+
+### Use AAB (Android App Bundle or just App Bundle)
+
+🚫Debug
```

Comment:
Hi @AntPolkanov 

Usually we use ❌ instead of 🚫

- [ ] 1. Can you fix all?
- [ ] 2. Please also add a space between emoji and text

Reply:
Done